airy.go

```diff
@@ -0,0 +1,29 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "github.com/gonum/mathext/internal/amos"
+
+// Airy returns the value of the Airy function at z. The Airy function here,
+// Ai(z), is one of the two linearly independent solutions to
+//  y'' - y*z = 0.
+// See http://mathworld.wolfram.com/AiryFunctions.html for more detailed information.
+func Airy(z complex128) complex128 {
+	id := 0
+	kode := 1
+	air, aii, _ := amos.Zairy(real(z), imag(z), id, kode)
+	return complex(air, aii)
+}
+
+// AiryDeriv returns the derivative of the value of the Airy function at z. The
```

Comment:
`// AiryDeriv returns the value of the derivative of the Airy function at z.` or something similar other than `derivative of the value`